molecule/update_cartridge/converge.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Bootstrap cluster with Cartridge 2.3.0 on all instances
+  hosts: cluster
+  roles:
+    - role: ansible-cartridge
+  become: true
+  become_user: root
+  gather_facts: false
+  vars:
+    cartridge_scenario_name: bootstrap_cluster
+    cartridge_enable_tarantool_repo: true
+    cartridge_configure_systemd_unit_files: true
+    cartridge_configure_tmpfiles: true
+    cartridge_install_tarantool_for_tgz: true
```

Comment:
What about to add comment that this variables is true by default?

Reply:
For what? This flags are set to false in inventory to rewrite them only in first play.